samples/logging_s3/pipeline_log_to_s3_by_fluentd_recommend/pipeline-logs-fluentd-s3.yaml

```diff
@@ -0,0 +1,134 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: fluentd
+  namespace: kube-system
+
+---
+apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: ClusterRole
+metadata:
+  name: fluentd
+  namespace: kube-system
+rules:
+- apiGroups:
+  - ""
+  resources:
+  - pods
+  - namespaces
+  verbs:
+  - get
+  - list
+  - watch
+
+---
+kind: ClusterRoleBinding
+apiVersion: rbac.authorization.k8s.io/v1beta1
+metadata:
+  name: fluentd
+roleRef:
+  kind: ClusterRole
+  name: fluentd
+  apiGroup: rbac.authorization.k8s.io
+subjects:
+- kind: ServiceAccount
+  name: fluentd
+  namespace: kube-system
+
+---
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: fluentd-s3
+  namespace: kube-system
+  labels:
+    k8s-app: fluentd-s3
+spec:
+  selector:
+    matchLabels:
+      k8s-app: fluentd-s3
+  template:
+    metadata:
+      labels:
+        k8s-app: fluentd-s3
+    spec:
+      serviceAccount: fluentd
+      serviceAccountName: fluentd
+      containers:
+      - name: fluentd-s3
+        image: fenglixa/pipeline-logs-s3:latest
```

Comment:
@Tomcli @fenglixa can we use a generic docker id for these images?

Reply:
@fenglixa I pushed the same image to `aipipeline/pipeline-logs-s3:latest` and also gave you permission to push any new update.

Reply:
Changed the docker image to `aipipeline/pipeline-logs-s3:latest`